NGitLab/PublicAPI.Unshipped.txt

```diff
@@ -1358,7 +1333,7 @@ NGitLab.Models.Event.TargetIId.get -> long?
 NGitLab.Models.Event.TargetIId.set -> void
 NGitLab.Models.Event.TargetTitle.get -> string
 NGitLab.Models.Event.TargetTitle.set -> void
-NGitLab.Models.Event.TargetType.get -> NGitLab.DynamicEnum<NGitLab.Models.EventTargetType>
+NGitLab.Models.Event.TargetType.get -> NGitLab.DynamicEnum<NGitLab.Models.EventTargetType>?
```

Comment:
I think a custom converter could map `null` to `EventTargetType.None` to avoid the breaking change

Reply:
Done